### ISSUE TITLE
[ENHANCEMENT] Position bed after finishing print in a way that it doesn't cover LCD

### ISSUE THREAD

**Issue by arekm** [prusa3d/Prusa-Firmware-Buddy#3785]
**Printer type** - MK3.5
**Firmware** - 5.2.0

**Is your suggested improvement related to an existing problem? Please describe.**

When MK3.5 printer finishes print it moves bed to the front for easier print sheet removing from the printer.

Unfortunately with bigger LCD (in comparison to MK3S+) is goes too far into front in a such way that it starts to make top of the LCD not visible to the user.

No idea if that's MK3.5 thing or if it also affect MK4.

![mk3 5-bad-bed](https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/50869/1ed49296-7b4c-44b4-b0c6-0f4412d124b4)

**Describe the expected functionality**

Don't move bed fully to front so it won't cover LCD. Like on this photo (bed was manually moved).

![mk3 5-bad-better](https://github.com/prusa3d/Prusa-Firmware-Buddy/assets/50869/0e852270-8737-4484-9763-cd7adc365b50)

**Comment by magistersmax** [comment]
This isn't a firmware issue, it's a gcode issue.  

You can fix this yourself by editing the end gcode in PrusaSlicer, a configuration update would be the way to roll it out to the community..

**Comment by magistersmax** [comment]
For some more detail and how to fix this yourself, in PrusaSlicer go to the Custom G-code page, and look for this under End G-code:

G1 X241 Y201 F3600 ; park

That is the line that sets the position of the extruder/bed after the print is complete.  The Xxxx and Yyyy sets the X and Y position in mm.  I like my extruder to be on the left side of the axis and I agree that the bed should be back a bit more, so I just changed that line to this:

G1 X10 Y180 F3600 ; park

Save it as a user preset and it'll carry over to the next things you slice.

**Comment by arekm** [comment]
Ok, Prusa devs, please move this end_gcode issue to https://github.com/prusa3d/PrusaSlicer-settings repository

**Comment by Tupson444** [comment]
> For some more detail and how to fix this yourself, in PrusaSlicer go to the Custom G-code page...

Actually, that would not work in this case, because the printer will override the gcode (that is, it will execute the gcode, then it will move the bed on the default position, so the end gcode will not have an effect). This was reported here: #3790

> This isn't a firmware issue, it's a gcode issue.

While this indeed is a gcode issue, it cannot be solved without first solving the firmware issue (#3790).

**Comment by magistersmax** [comment]
I stand corrected.  It does seem like #3790 is the actual issue here.

**Comment by Acidham** [comment]
Any update before it gets closed?

**Comment by danopernis** [comment]
This is certainly possible to fix with proper configuration of the Marlin NOZZLE_PARK_FEATURE

**Comment by Acidham** [comment]
Agreed, and even better solution to add it to config. When the LCD is assembled to the printer, the heat bed is hiding the screen.

**Comment by Tupson444** [comment]
> NOZZLE_PARK_FEATURE

This is it then, probably. The printer parks nozzle (print head) every time after finishing print.

This should be changed that it only parks if such a command is actually present in the gcode file (`G27` ?). Hardcoded commands hinder customizability.

**Comment by Tupson444** [comment]
I can confirm that this was fixed in the meantime: the printer now respects end code commands (at least for X & Y position, Z is still overridden, but that is not a problem). Tested on MK4 and Core One (6.4.0 RC).

**Comment by 3d-gussner** [comment]
@Tupson444 Thanks for the feedback.
